Sprint 1/Estimation Documents/MoSCoW.md

```diff
@@ -0,0 +1,30 @@
+#App:
+    **Must:**
+        Have a questionaire to enter data
+        Send data to the server
+        Work on iOS devices
+    **Should:**
+        Sanitise SQL inputs (or otherwise defend against harm)
```

Comment:
This part should probably be at the server side as well because users can by pass the app to send malicious sql inputs directly